### DISCUSSION TITLE
Video autoplay

### DISCUSSION THREAD

**Discussion by moon95light** [cdnjs/cdnjs#14221] [General] [answered]
Hello
I hope you are doing well.
I am using the vidoe cdn min js file to display the video file in my web app.
`<script src="https://cdnjs.cloudflare.com/ajax/libs/video.js/8.6.1/video.min.js"
    integrity="sha512-19kPqSYAN3EiTxmPPFeInu0KiE6ZpYntGctkdtc2LGShfM1QcZQA2O8y25og2lufK5bE2gSnYn5PO2+9Iex4Bg=="
    crossorigin="anonymous" referrerpolicy="no-referrer"></script>
`
I wanna set the video autoplay when load this page.
How to fix this problem?
Please let me know the good way and Help me.
Thank you for your time and consideration.

**Comment by MattIPv4** [comment, marked as answer]
👋 While cdnjs hosts the library you are using, we are not the authors of the library and cannot provide support for it. You'll want to reach out in an appropriate support forum for the library you are using itself.

https://cdnjs.com/libraries/video.js has a link to the GitHub repository where you can potentially ask for help: https://github.com/videojs/video.js
